src/core/qfieldcloudconnection.cpp

```diff
@@ -181,14 +182,16 @@ void QFieldCloudConnection::login()
       {
         emit loginFailed( tr( "Timeout error, please retry" ) );
       }
-      else if ( httpCode == 400 )
+      else if ( httpCode == 400 || httpCode == 401 )
       {
-        emit loginFailed( tr( "Wrong username or password" ) );
-      }
-      else if ( httpCode == 401 )
-      {
-        emit loginFailed( tr( "Session expired" ) );
-        invalidateToken();
```

Comment:
Is it ok to skip the `invalidateToken`?

Reply:
@suricactus , yes, it's invalidated by the get() function (https://github.com/opengisch/QField/blob/master/src/core/qfieldcloudconnection.cpp#L380-L387)

Reply:
Ok, thanks!